### DISCUSSION TITLE
Search for a keyword from the closest or most relevant location to your expectations.

### DISCUSSION THREAD

**Discussion by admiral504** [omkarcloud/google-maps-scraper#46] [Ideas]
Hello OmkarCloud, here's my suggestion for the Google Maps scraper:

The current google-maps-scraper configuration to search for digital marketers in Kathmandu looks like this:

`queries = [
  {
    "keyword": "digital marketers in kathmandu",
  },
]`

We should upgrade the config to this:

`[
  {
    "keyword": "digital marketers", 
    "coordinate": "27.714581771025006, 85.32681929134418"
  }
]`

The coordinate will construct the search URL like this:

`https://www.google.com/maps/search/digital+marketers/@27.714581771025006,85.32681929134418,13z/data=!3m1!4b1?hl=vi-VN&entry=ttu`

Google generates search results based on the searcher's keywords and their location. This is why it's advisable to search for a keyword from the closest or most relevant location to your expectations.

**Comment by Chetan11-dev** [comment]
Hi Admiral,
Thank you for your suggestion. At this point of time, I'm not planning to add this feature. 

Most importantly because there hasn't been demand for it from the users.

However, if more users express interest in this feature in the future, I will certainly reconsider my decision.

**Comment by Chetan11-dev** [comment]
Interested to learn about you Admiral, Should [we meet](https://www.omkar.cloud/l/meet-chetan/)?
